subprojects/docs/src/docs/userguide/jvm/java_library_plugin.adoc

```diff
@@ -278,17 +278,6 @@ Gradle will automatically put a Jar of your dependencies on the module path, ins
 In the following, some more details about defining Java modules and how that interacts with Gradle's dependency management are described.
 You can also look at a link:../samples/sample_java_modules_multi_project.html[ready made example] to try out the Java Module support directly.
 
-[NOTE]
```

Comment:
I think there should be a snippet here showing how to turn it off. AFAICT it does not exist anywhere.

Reply:
I was a bit reluctant since no one should actually have to do that. Either you do not have a module, then the inference won't be triggered anyway. Or you have a module - which then won't compile if you disable the inference. You might only need this if you really want to use one of the pre 6.4 'hacky' 3rd party plugins.

But you are probably right that it should be documented somewhere. I added a paragraph with a snippet. Let me know what you think.

Reply:
Looks good to me, thanks for adding it.